src/foam/u2/crunch/CrunchController.js

```diff
@@ -426,7 +426,7 @@ foam.CLASS({
       wizardController.wizardlets[wi].wao = this.TopicWAO.create({
         delegate: wizardController.wizardlets[wi].wao
       });
-      wizardController.onDetach(wizardController.wizardlets[wi].wao.saving.sub(
+      wizardController.onDetach(wizardController.wizardlets[wi].wao.saving?.sub(
```

Comment:
Huh, wonder how this happens since we override the WAO in the line above this to one that does have saving

Reply:
@KernelDeimos

Reply:
<img width="1440" alt="Screen Shot 2023-02-03 at 9 30 51 AM" src="https://user-images.githubusercontent.com/10802216/216648784-8de9148d-8ccf-4c01-be91-62d3e44bb003.png">
<img width="1440" alt="Screen Shot 2023-02-03 at 9 31 05 AM" src="https://user-images.githubusercontent.com/10802216/216648809-d5315a9a-5df2-4ab0-b4c8-832aedfa2158.png">
I was seeing this - but I just tested it again .... and ...not seeing what I'd expect - so hold on this pr